playbooks/openshift/roles/deploy/tasks/set_docker_registry.yml

```diff
@@ -0,0 +1,22 @@
+- name: comment out current registry lines
```

Comment:
I didn't have to do this on the OSE side I just point it at pulp like you do in your defaults.  I ended up not using this playbook